drizzle-kit/src/cli/commands/migrate.ts

```diff
@@ -391,6 +395,150 @@ export const prepareAndMigrateMysql = async (config: GenerateConfig) => {
 	}
 };
 
+// Not needed for now
+function mySingleStoreSchemaSuggestions(
```

Comment:
The "my" is extra here

Reply:
Deleted this function because it wasn't used anywhere